website/www/site/content/en/blog/pattern-match-beam-sql.md

```diff
@@ -0,0 +1,167 @@
+---
+title:  "Pattern Matching with Beam SQL"
+date:   2020-08-27 00:00:01 +0800
+aliases:
+  - /blog/2020/08/27/pattern-match-beam-sql.html
+categories:
+  - blog
+authors:
+  - Mark-Zeng
```

Comment:
Better to your real name and email :)

Reply:
Would using github handle automatically render into my real name?

Reply:
Ah Yes you are right.